user/config/config_gotls.go

```diff
@@ -103,15 +104,25 @@ func (gc *GoTLSConfig) Check() error {
 // which are actually mounted via uprobe on these addresses.
 func (gc *GoTLSConfig) findRetOffsets(symbolName string) ([]int, error) {
 	var err error
-	var goSymbs []elf.Symbol
-	goSymbs, err = gc.goElf.Symbols()
-	if err != nil {
-		return nil, err
+	var allSymbs []elf.Symbol
+
+	goSymbs, _ := gc.goElf.Symbols()
+	if len(goSymbs) > 0 {
+		allSymbs = append(allSymbs, goSymbs...)
+	}
+
+	goDynamicSymbs, _ := gc.goElf.DynamicSymbols()
```

Comment:
in stripped compiled files ,  there are also no symbols in the `DynamicSymbols` section.

Reply:
@cfc4n , Thanks for commenting. I had shared more info in the #428 , for the reason behind including dynamic symbols